src/game.zig

```diff
@@ -717,25 +727,25 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 						movementSpeed = @max(movementSpeed, 32)*KeyBoard.key("forward").value;
 						movementDir += forward*@as(Vec3d, @splat(32*KeyBoard.key("forward").value));
 					} else {
-						movementSpeed = @max(movementSpeed, 8)*KeyBoard.key("forward").value;
-						movementDir += forward*@as(Vec3d, @splat(8*KeyBoard.key("forward").value));
+						movementSpeed = @max(movementSpeed, MaxSpeedRunning)*KeyBoard.key("forward").value;
+						movementDir += forward*@as(Vec3d, @splat(MaxSpeedRunning*KeyBoard.key("forward").value));
 					}
 				} else {
-					movementSpeed = @max(movementSpeed, 4)*KeyBoard.key("forward").value;
-					movementDir += forward*@as(Vec3d, @splat(4*KeyBoard.key("forward").value));
+					movementSpeed = @max(movementSpeed, MaxSpeedWalking)*KeyBoard.key("forward").value;
+					movementDir += forward*@as(Vec3d, @splat(MaxSpeedWalking*KeyBoard.key("forward").value));
 				}
 			}
 			if(KeyBoard.key("backward").value > 0.0) {
-				movementSpeed = @max(movementSpeed, 4)*KeyBoard.key("backward").value;
-				movementDir += forward*@as(Vec3d, @splat(-4*KeyBoard.key("backward").value));
+				movementSpeed = @max(movementSpeed, MaxSpeedWalking)*KeyBoard.key("backward").value;
+				movementDir += forward*@as(Vec3d, @splat(-MaxSpeedWalking*KeyBoard.key("backward").value));
 			}
 			if(KeyBoard.key("left").value > 0.0) {
-				movementSpeed = @max(movementSpeed, 4*KeyBoard.key("left").value);
-				movementDir += right*@as(Vec3d, @splat(4*KeyBoard.key("left").value));
+				movementSpeed = @max(movementSpeed, MaxSpeedWalking*KeyBoard.key("left").value);
+				movementDir += right*@as(Vec3d, @splat(MaxSpeedWalking*KeyBoard.key("left").value));
 			}
 			if(KeyBoard.key("right").value > 0.0) {
-				movementSpeed = @max(movementSpeed, 4*KeyBoard.key("right").value);
-				movementDir += right*@as(Vec3d, @splat(-4*KeyBoard.key("right").value));
+				movementSpeed = @max(movementSpeed, MaxSpeedWalking*KeyBoard.key("right").value);
+				movementDir += right*@as(Vec3d, @splat(-MaxSpeedWalking*KeyBoard.key("right").value));
```

Comment:
While I appreciate that you try to clean up other code along the way, I don't think just renaming the numbers will suffice.
Furthermore this section is already edited by another open PR #705 and it would be best to avoid unnecessary conflicts.

Reply:
Actually, the main reason was because I wanted the FOV to only change when you go above walking speed and be maxed out at sprinting speed.
The walking speed is 4 and sprinting speed is 8. But I didn't want the FOV code to break if anyone would change the numbers here. So, I made them a constant and both the movement speed code and the fov code use them..

Reply:
Yeah, but if you for example have enabled hyperspeed, then the max sprinting speed is 32. Shouldn't this give more fov boost?

Reply:
I assumed that was kind of a debug thing :D 
I'll replace the clamped formula with a curved function instead, then I also no longer need these modifications here

Reply:
No, it's a keybind, `G`+`H` gives you the largest speed.
It's very useful if you want to fly to a distant location in creative mode.